src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -96,11 +94,11 @@ def erosion_coeff(ws, season):
     `rain_cool` instead.
     """
 
-    avg_warm = np.mean([w.rain_warm for w in ws])
-    avg_cool = np.mean([w.rain_cool for w in ws])
+    avg_warm = float(sum([w.rain_warm for w in ws])) / len(ws)
+    avg_cool = float(sum([w.rain_cool for w in ws])) / len(ws)
```

Comment:
Not sure it matters much since the precision's probably enough but it looks as if the numpy `mean` method uses `float64`s instead of `float`s.